### PR TITLE
VIH-7394 automate ejud joh demo

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7394

### Change description ###
- Send a generic email to ejud joh when the case type is 'Generic'
- Excluding sending generic emails to a Hearing Type 'Automated Test'

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs [hunk 1]
```diff
@@ -130,14 +131,14 @@ public async Task should_send_confirmation_email_to_all_participants_except_judg
                     Times.Exactly(3));
         }
 
-        [Test]
+       [Test]
         public async Task should_not_send_confirmation_email_when_hearing_is_generic_case_type()
```
Comment (mulvaneyjHMCTS) [inline AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs:135]: Name of test needs to be changed?

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.html [hunk 2]
```diff
@@ -114,8 +114,8 @@ <h2 class="govuk-heading-m">Time and location</h2>
           <span id="hearingDateWeekend-error" class="govuk-error-message"> Select a date </span>
         </div>
 
-        <div *ngIf="addHearingDateControl.errors?.weekend || addHearingDateControl.errors?.publicHoliday" class="alert alert-danger">
-          <span id="hearingDateWeekend-error" class="govuk-error-message"> Select a working day </span>
+        <div *ngIf="addHearingDateControl.errors?.weekend" class="alert alert-danger">
+          <span id="hearingDateWeekend-error" class="govuk-error-message"> Select a day from Monday to Friday </span>
```
Comment (mulvaneyjHMCTS) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.html:118]: What is this for?

Comment (PrasannaDommalapati) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.html:118]: this one is merged from another story to exclude the hearing dates should not be on weekends/band holidays

File: AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs (hunk 1 above)
Comment (PrasannaDommalapati) [reply AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs]: changed

**Review by mulvaneyjHMCTS — DISMISSED** [review]

**Review by mulvaneyjHMCTS — APPROVED** [review]

**Review by mulvaneyjHMCTS — APPROVED** [review]

**Pull request merged**